server/src/main/java/org/opensearch/cluster/service/MasterService.java

```diff
@@ -913,7 +921,7 @@ private ClusterTasksResult<Object> executeTasks(TaskInputs taskInputs, ClusterSt
                     "failed to execute cluster state update (on version: [{}], uuid: [{}]) for [{}]\n{}{}{}",
                     previousClusterState.version(),
                     previousClusterState.stateUUID(),
-                    taskInputs.summary,
+                    taskSummary,
```

Comment:
We are always passing short summary. Since, we already computed short summary at beginning of run, so i am passing that value instead of again calling taskInputs.taskSummaryGenerator.apply(false).

Reply:
Passed shortSummary, i got confused with a different flow.